src/main/kotlin/com/project/twiliospring/service/impl/F2AServiceImpl.kt

```diff
@@ -0,0 +1,42 @@
+package com.project.twiliospring.service.impl
+
+import com.project.twiliospring.domain.F2ARecord
+import com.project.twiliospring.exception.F2ARecordNotFoundException
+import com.project.twiliospring.repository.F2ARepository
+import com.project.twiliospring.service.F2AService
+import com.project.twiliospring.service.MessengerService
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Profile
+import org.springframework.stereotype.Service
+import kotlin.random.Random
+
+@Service
+@Profile("!twilio")
```

Comment:
probably we can pass auth method in controller and use Custom or Twilio F2A service